Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:24
+FROM registry.fedoraproject.org/fedora:25
```

Comment:
When I initially wrote this commit, F26 wasn't released yet. I'll bump to `fedora:26` in the next PR once I have have chance to test it out.